From dea0155b131b55cdc1bb3bc867ad53c1782fa724 Mon Sep 17 00:00:00 2001
From: Til Kaiser <mail@tk154.de>
Date: Fri, 7 Feb 2025 14:52:35 +0100
Subject: [PATCH 3/3] dev_mgt: include missing function declaration

The dev_mgt fails to build with the following error:

tools_dev_types.c: In function 'dm_get_device_id_inner':
tools_dev_types.c:695:13: error: implicit declaration of function 'read_device_id'; did you mean 'dm_get_device_id'? [-Wimplicit-function-declaration]
  695 |         if (read_device_id(mf, &dword) != 4) {
      |             ^~~~~~~~~~~~~~
      |             dm_get_device_id

This commit includes mtcr_ul_com.h (where read_device_id
is defined) into tools_dev_types.c to fix the error.

Signed-off-by: Til Kaiser <mail@tk154.de>
---
 dev_mgt/tools_dev_types.c | 1 +
 1 file changed, 1 insertion(+)

--- a/dev_mgt/tools_dev_types.c
+++ b/dev_mgt/tools_dev_types.c
@@ -48,6 +48,7 @@
 #include <tools_layouts/reg_access_hca_layouts.h>
 #include "tools_dev_types.h"
 #include "mflash/mflash_types.h"
+#include "mtcr_ul/mtcr_ul_com.h"
 
 enum dm_dev_type {
     DM_UNKNOWN = -1,
